src/components/cards/cards.comp.tsx

```diff
@@ -64,6 +64,8 @@ export const Cards = ({ items, fields, callbacks, customActions, customLabels, p
         return <a href={url} target="_blank" rel="noopener noreferrer">{origField.urlLabel || value}</a>;
       case 'colorbox':
         return <div className="colorbox" style={{ backgroundColor: value }}></div>;
+      case 'json':
```

Comment:
Does the support of `json` type is part of this pull request? Maybe it should be seperated?

Reply:
Ah yeah, good call.